.vib/mariadb-galera/goss/goss.yaml

```diff
@@ -0,0 +1,40 @@
+file:
+  /bitnami/mariadb/data/{{ .Vars.db.name }}:
+    mode: "2700"
+    filetype: directory
+    exists: true
+  /opt/bitnami/mariadb/.bootstrap:
+    mode: "2777"
+    filetype: directory
+    exists: true
+  /opt/bitnami/mariadb/conf/my.cnf:
+    mode: "0664"
+    filetype: file
+    exists: true
+    contains:
+      - "basedir=/opt/bitnami/mariadb"
+      - /wsrep_provider=.*libgalera_smm.so/
+      - "wsrep_cluster_name={{ .Vars.galera.name }}"
+command:
+  create-table-root:
+    exec: mariadb -h mariadb-galera -P {{ .Vars.service.ports.mysql }} -u {{ .Vars.rootUser.user }} -p'{{ .Vars.rootUser.password }}' {{ .Vars.db.name }} -e 'DROP TABLE IF EXISTS TEST; create table TEST(test_id int auto_increment, test_value int, primary key(test_id)); INSERT INTO TEST (TEST_VALUE) VALUES (1989);SELECT * FROM TEST'
+    exit-status: 0
+    stdout:
+    - 1989
```

Comment:
Can we that value add it as variable? To me is weird have it harcoded twice in the test.